test/uk/gov/hmrc/plasticpackagingtax/registration/controllers/helpers/LiabilityLinkHelperSpec.scala

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.plasticpackagingtax.registration.controllers.helpers
+
+import base.PptTestData
+import base.unit.ControllerSpec
+import org.mockito.Mockito.when
+import org.scalatest.matchers.must.Matchers.convertToAnyMustWrapper
+import play.api.mvc.Headers
+import uk.gov.hmrc.http.HeaderNames
+import uk.gov.hmrc.plasticpackagingtax.registration.controllers.routes
+import uk.gov.hmrc.plasticpackagingtax.registration.forms.LiabilityWeight
+import uk.gov.hmrc.plasticpackagingtax.registration.models.registration.LiabilityDetails
+import uk.gov.hmrc.plasticpackagingtax.registration.models.request.JourneyRequest
+
+class LiabilityLinkHelperSpec extends ControllerSpec {
+
+  private val helper =
+    LiabilityLinkHelper(appConfig = config)
+
+  "Backlink for liability weight page" should {
+
+    "when liability weight is more than minimum weight" in {
```

Comment:
These tests don't read very well when combined. Perhaps something like the following might work better?
```
"The Liability Link Helper" should {
  "provide a back link to the liability weight page" when {
    "weight exceeds threshold" in {
    }
  }
}
```

Reply:
Done